AWSS3Tests/AWSS3TransferUtilityTests.swift

```diff
@@ -2322,7 +2323,16 @@ class AWSS3TransferUtilityTests: XCTestCase {
                 
                 if let HTTPResponse = task.response {
                     XCTAssertEqual(HTTPResponse.statusCode, 200)
-                    
+
+                    let timeout = DispatchTime.now() + .seconds(2)
+                    while(!AWSS3TestHelper.checkIfObjectIsPresent(key, bucket: generalTestBucket)) {
+                        if (DispatchTime.now() > timeout )  {
+                            XCTFail("Could not verify uploaded key \(key)")
+                            break;
```

Comment:
Consider a `return` instead of a `break`, since the rest of the test conditions will fail if the object isn't present.

Reply:
Fixed